Assignment2/LuisLopezPi-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,36 @@
+{
+	"@context": {
+		"fiupm": "http://www.fiupm.es/alumnos/cuarto/",
+		"name": "fiupm:name",
+		"hometown": "fiupm:hometown",
+		"age": "fiupm:age",
+		"almaMater": "fiupm:alma mater",
+		"mother": "fiupm:mother",
+		"father": "fiupm:father",
+		"subjects": "fiupm:subjects",
+		"description": "subjects:description",
+		"nameS": "subjects:nameS",
+		"teacher": "subjects:teacher"
```

Comment:
These subject attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes.
You don't need to change them, but remember to provide good definitions for the future.